### PR TITLE
data structures test pass

### PR DESCRIPTION
**Pull request by elsaalarcon**

### CODE REVIEW

File: spec/data_structures_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+describe 'data_structures' do
+  it "array cero will be cat" do
+    expect(animals[0]).to eq 'cat'
+  end
+
+  it "array one will be dog" do
+    expect(animals[1]).to eq 'dog'
+  end
+end
+
```
Comment (geocodinglife) [inline spec/data_structures_spec.rb]: Remove extra lines

File: two/data_structures.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+def animals
+  ["cat", "dog", "wolf"]
+end
+
```
Comment (geocodinglife) [inline two/data_structures.rb]: Remove extra lines

File: two/data_structures.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+def animals
+  ["cat", "dog", "wolf"]
```
Comment (geocodinglife) [inline two/data_structures.rb]: Pleas change doble "" to single ones ''

File: spec/data_structures_spec.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+require_relative '../two/data_structures.rb'
+
+describe 'data_structures' do
+  it "array cero will be cat" do
```
Comment (geocodinglife) [inline spec/data_structures_spec.rb]: Please change doble quotes  "" to single ones ''

File: spec/data_structures_spec.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    expect(animals[0]).to eq 'cat'
+  end
+
+  it "array one will be dog" do
```
Comment (geocodinglife) [inline spec/data_structures_spec.rb]: Please change doble quotes  "" to single ones ''

File: two/data_structures.rb (hunk 2 above)
Comment (elsaalarcon) [reply two/data_structures.rb]: done

File: spec/data_structures_spec.rb (hunk 1 above)
Comment (elsaalarcon) [reply spec/data_structures_spec.rb]: done

File: two/data_structures.rb (hunk 3 above)
Comment (elsaalarcon) [reply two/data_structures.rb]: done

File: spec/data_structures_spec.rb (hunk 4 above)
Comment (elsaalarcon) [reply spec/data_structures_spec.rb]: done

**Review by geocodinglife — APPROVED** [review]
Looks good!

**Pull request merged**